flink-core/pom.xml

```diff
@@ -180,6 +180,13 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
Mockito is intentionallly excluded from Flink and adding this dependency to flink-core sends a wrong message. Please check if you can avoid using it.
https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#avoid-mockito---use-reusable-test-implementations

Reply:
Done.

I had to use a separate project, because tests in flink-core depend on concrete logging implementation, which doesn't exhibit the behavior required to hit the new code path.